lib/bamboo/test.ex

```diff
@@ -146,7 +144,7 @@ defmodule Bamboo.Test do
   Checks whether an email was delivered.
 
   Must be used with the `Bamboo.TestAdapter` or this will never pass. In case you
-  are delivering from another process, the assertion waits up to 100ms before
+  are delivering from another process, the assertion waits up to 150ms before
```

Comment:
At the time I ran the tests, 100ms seemed to not be enough. But I ran the tests now 3 times in a row with the 100ms and they seem fine.

Changed back to 100ms.